.github/workflows/npm-cd.yml

```diff
@@ -16,7 +17,23 @@ concurrency:
     cancel-in-progress: true
 
 jobs:
+    start-self-hosted-runner:
+      runs-on: ubuntu-latest
+      steps:
+        - name: Checkout
+          uses: actions/checkout@v4
+          with:
+              submodules: "true"
+        - name: Start self hosted EC2 runner
+          uses: ./.github/workflows/start-self-hosted-runner
+          with:
+              aws-access-key-id: ${{ secrets.AWS_EC2_ACCESS_KEY_ID }}
```

Comment:
if possible to move these into the script instead of taking them from the env, it would be nice.

Reply:
Agreed, but sadly can't be done